### DISCUSSION TITLE
User Switching

### DISCUSSION THREAD

**Discussion by linjiaen** [filamentphp/filament#12888] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.77

### How can we help you?

As a newcomer, I have a scenario where I am an administrator. There is a button in the user list, and when I click on it, I enter the user's panel to view their data and perform operations on their behalf. In essence, the logged-in user has been switched.

**Comment by dissto** [comment, marked as answer]
https://filamentphp.com/plugins/joseph-szobody-impersonate ?

**Reply by linjiaen** [reply]
Thank you very much for your help
